library/rust

```diff
@@ -3,22 +3,22 @@
 Maintainers: Steven Fackler <sfackler@gmail.com> (@sfackler)
 GitRepo: https://github.com/rust-lang-nursery/docker-rust.git
 
-Tags: 1.26.2-stretch, 1-stretch, 1.26-stretch, stretch, 1.26.2, 1, 1.26, latest
+Tags: 1.27.0-stretch, 1.27.0
```

Comment:
I assume the dropping and not adding generic tags like `1-stretch` and `1.27` was on purpose.

Did you also mean to drop `latest`? It is the tag pulled if a user just does `docker pull rust`. As is, they will continue to get `1.26.2-stretch` if they forget to specify a tag.

Reply:
Gah, nope that's my bad. Will fix.

Reply:
Done!